e2e/test/scenarios/collections/uploads.cy.spec.js

```diff
@@ -161,6 +163,83 @@ describeWithSnowplow(
   },
 );
 
+describeEE("permissions", () => {
+  it("should not snow you upload buttons if you are a sandboxed user", () => {
+    restore("postgres-writable");
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+    enableUploads("postgres");
+
+    const now = Date.now();
+
+    // We need up upload a file to the writable DB so that we can give
+    // out sandboxed user access to *something* on the upload DB
+    cy.readFile(`e2e/support/assets/${validTestFiles[0].fileName}`).then(
```

Comment:
If I got it right, the fix is not releated to uploads, it is about having a write permission to a database so we can test just it. This spec looks like an overkill because it uploads a csv, but it does not make it different from a regular `postgres-12` database snapshot selected as the upload target. I would suggest simplifying it by removing unnecessary steps